### DISCUSSION TITLE
apache skywalking storage

### DISCUSSION THREAD

**Discussion by kannan84240** [apache/skywalking#9124] [Q&A] [answered]
I'm using apache skywalking 8 version, 
Q1: can i able to take previous some one day report ?
Q2: is this permanent storage, if not means how can i store all reports permanently 

Thanks in advance .
![Skywalking_report ](https://user-images.githubusercontent.com/73342985/169964682-18ec8439-6bc7-4a12-865d-4d5aa5452da8.png)

**Comment by wu-sheng** [comment, marked as answer]
You should learn TTL concept. https://skywalking.apache.org/docs/main/latest/en/setup/backend/ttl/

**Reply by kannan84240** [reply]
okay thanks, how can I backup all reports

**Reply by wu-sheng** [reply]
I think you mixed SkyWalking with other systems. We are an APM, not a BI system.

**Reply by kannan84240** [reply]
Oh, is must monitoring tool and agent installed in same application server ?

**Reply by wu-sheng** [reply]
What is `monitoring tool`? 

The agent is in your application process, so, one way or another, yes.

**Reply by kannan84240** [reply]
I'm asking  apache-skywalking-apm-8.9.1.tar.gz and apache-skywalking-java-agent-8.10.0.tgz and my application all are in same server to install?

**Reply by wu-sheng** [reply]
Reading the doc before asking questions, https://skywalking.apache.org/docs/main/v9.0.0/en/concepts-and-designs/overview/#architecture

This is an open-source community, not a customer service.

**Reply by kannan84240** [reply]
Ok thanks
